Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM nvidia/cuda:11.1.1-cudnn8-devel-ubuntu20.04
+# MLCommons Training Algorithms Dockerfile
+#
+# docker build https://github.com/mlcommons/algorithmic-efficiency.git
 
-RUN apt-get update && \
-    apt-get install -y bash \
+FROM nvidia/cuda:11.5.1-cudnn8-devel-ubuntu20.04
```

Comment:
We might need to use CUDA 11.6 or 11.7, because it seems like the most recent versions of Jax do not support 11.5 anymore due to their use of NCCL 2.12.12. See issue #105 for more details on the relevance of updating the Jax version and the potential compatibility issue.

Reply:
great catch, updated to the 11.7 version of the same base image